src/org/labkey/test/util/DataClassHelper.java

```diff
@@ -0,0 +1,286 @@
+        Set<String> expectedNames = _fields.stream().map(FieldDefinition::getName).collect(Collectors.toSet());
+        expectedNames.add("Name");
+        expectedNames.add("Description");
+        assertEquals("Fields in data class not as expected.", expectedNames, actualNames);
+    }
+
+    public DataRegionTable getDataClassesList()
+    {
+        return new DataRegionTable.DataRegionFinder(getDriver()).withName(DATA_CLASS_DATA_REGION_NAME).find();
+    }
+
+    public DataRegionTable getDataClassDataRegionTable()
+    {
+        return new DataRegionTable.DataRegionFinder(getDriver()).withName("query").find();
+    }
+
+    public int getDataCount()
+    {
+        return getDataClassDataRegionTable().getDataRowCount();
+    }
+
+    private List<String> getDataClassFields()
+    {
+        return getDataClassDataRegionTable().getColumnNames();
+    }
+
+    public void insertRow(Map<String, String> fieldValues)
+    {
+        getDataClassDataRegionTable()
+                .clickInsertNewRow();
+        for (Map.Entry<String, String> fieldValue : fieldValues.entrySet())
+        {
+            setFormElement(Locator.name("quf_" + fieldValue.getKey()), fieldValue.getValue());
+        }
+        clickButton("Submit");
+    }
+
+    public void bulkImport(File dataFile)
+    {
+        fileImport(dataFile, IMPORT_OPTION);
+    }
+
+    public void mergeImport(File dataFile)
+    {
+        fileImport(dataFile, MERGE_OPTION);
+    }
+
+    private void fileImport(File dataFile, String importOption)
+    {
+        DataRegionTable drt = getDataClassDataRegionTable();
+        TestLogger.log("Adding data from file");
+        ImportDataPage importDataPage = drt.clickImportBulkData();
+        importDataPage.setFile(dataFile);
+        if (MERGE_OPTION.equals(importOption))
+            importDataPage.setFileMerge(true);
+        else if (UPDATE_OPTION.equals(importOption))
+            importDataPage.setFileInsertOption(true);
+        importDataPage.submit();
+    }
+
+    public void bulkImport(String tsvData)
+    {
+        startTsvImport(tsvData, IMPORT_OPTION)
+                .submit();
+    }
+
+    public void mergeImport(String tsvData)
+    {
+        startTsvImport(tsvData, DataClassHelper.MERGE_OPTION)
+                .submit();
+    }
+
+    public void bulkImport(List<Map<String, String>> data)
+    {
+        startTsvImport(data, IMPORT_OPTION)
+                .submit();
+    }
+
+    public void bulkImportExpectingError(List<Map<String, String>> data, String importOption)
+    {
+        startTsvImport(data, importOption)
+                .submitExpectingError();
+    }
+
+    public void mergeImport(List<Map<String, String>> data)
+    {
+        startTsvImport(data, DataClassHelper.MERGE_OPTION)
+                .submit();
+    }
+
+    public void mergeImportExpectingError(List<Map<String, String>> data)
+    {
+        startTsvImport(data, DataClassHelper.MERGE_OPTION)
+                .submitExpectingError();
+    }
```

Comment:
Same comment as above. 
Since these don't appear to be used you may also choose to remove these methods. Whoever writes the test cases that could use these methods can add them.